.happy/terraform/modules/sfn/main.tf

```diff
@@ -233,8 +232,7 @@ resource "aws_sfn_state_machine" "state_machine" {
           "BackoffRate": 5
       }, {
           "ErrorEquals": ["States.TaskFailed"],
-          "IntervalSeconds": 30,
-          "MaxAttempts": 1
+          "MaxAttempts": 0
```

Comment:
I was thinking we should leave the auto-retry until we go a while without issues or until it becomes a problem that we're retrying mostly jobs that keep failing

Reply:
good catch - this wasn't intentional and was a mistake made while rebasing 😅